docs/developer/apis/camino-node-apis/index-api.md

```diff
@@ -394,7 +396,7 @@ To get an X-Chain transaction by its index (the order it was accepted in), use I
 For example, to get the _second_ transaction (note that `"index":1`) accepted on the X-Chain, do:
 
 ```sh
-curl --location --request POST 'https://indexer-test.camino.foundation/ext/index/X/tx' \
+curl --location --request POST 'https://indexer-test.camino.network/ext/index/X/tx' \
```

Comment:
there is no such subdomain . maybe it should be https://api.network/ext/index/X/tx  ?

Reply:
I'll take a look and fix this in another PR